lib/feature_flagger/model.rb

```diff
@@ -12,26 +12,30 @@ def self.included(base)
       base.extend ClassMethods
     end
 
-    def rollout?(feature_key)
+    def rollout?(*feature_key)
+      feature_key.flatten!
       resource_name = self.class.rollout_resource_name
       Feature.new(feature_key, resource_name).fetch!
       FeatureFlagger.control.rollout?(feature_key, id, resource_name)
     end
 
-    def release!(feature_key)
+    def release!(*feature_key)
+      feature_key.flatten!
       resource_name = self.class.rollout_resource_name
       Feature.new(feature_key, resource_name).fetch!
       FeatureFlagger.control.release!(feature_key, id, resource_name)
     end
 
-    def unrelease!(feature_key)
+    def unrelease!(*feature_key)
+      feature_key.flatten!
```

Comment:
Sorry, what's the reason to add this line here? since `feature_key` is already an array inside this scope.